var/spack/repos/builtin/packages/adol-c/package.py

```diff
@@ -80,6 +82,11 @@ def configure_args(self):
                 '--enable-atrig-erf'
             )
 
+        if '~stdczero' in spec:
+            configure_args.append(
+                '--disable-stdczero'
+            )
```

Comment:
```suggestion
        configure_args.extend(self.enable_or_disable('stdczero'))
```

Reply:
Should I unify the handling of the existing configure args while I'm at at?

Reply:
@severinstrobl If you want to that would be great!